src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -1111,18 +1111,12 @@ void SpatialPooler::inhibitColumns_(vector<Real>& overlaps,
     density = min(density, (Real) 0.5);
   }
 
```

Comment:
They are passing now. Why wouldn't they pass without removing it? As far as I can tell, it's just extra unused data.